libs/common/src/sql-db-entities/join-tables/fee-record-payments-payment/fee-record-payments-payment.entity.ts

```diff
@@ -0,0 +1,29 @@
+import { Entity, ManyToOne, JoinColumn, PrimaryColumn, Index } from 'typeorm';
+import { FeeRecordEntity } from '../../fee-record';
+import { PaymentEntity } from '../../payment';
+
+@Entity('fee_record_payments_payment')
+export class FeeRecordPaymentJoinTableEntity {
+  @PrimaryColumn({ nullable: false })
+  @Index('IDX_7a9b7aa849fc3bb09d80fa1f81')
+  feeRecordId!: number;
+
+  @ManyToOne(() => FeeRecordEntity, (feeRecord) => feeRecord.id, {
+    eager: false,
+    onDelete: 'CASCADE',
+    onUpdate: 'CASCADE',
+  })
+  @JoinColumn({ name: 'feeRecordId', foreignKeyConstraintName: 'FK_7a9b7aa849fc3bb09d80fa1f812' })
+  feeRecord!: FeeRecordEntity;
+
+  @PrimaryColumn({ nullable: false })
+  @Index('IDX_23bbb10be5f2136a5a5086654e')
+  paymentId!: number;
+
+  @ManyToOne(() => PaymentEntity, (payment) => payment.id, {
+    eager: false,
+    onDelete: 'CASCADE',
```

Comment:
Could I just double check that we're wanting to delete the payment as well as the fee records on delete? I noticed that [this constraint in the migration has `NO ACTION`](https://github.com/UK-Export-Finance/dtfs2/blob/82b874c6e64e4b7b1a4e1f48da38d267278e572a/libs/common/src/sql-db-connection/migrations/1716990167763-AddPaymentTable.ts#L40)

Reply:
Ah yes, thanks